app/src/main/java/com/hedvig/app/feature/keygear/KeyGearValuationViewModel.kt

```diff
@@ -38,19 +40,26 @@ class KeyGearValuationViewModelImpl(
     ) {
         viewModelScope.launch {
             val result = runCatching { repository.updatePurchasePriceAndDateAsync(id, date, price) }
+            if (result.isFailure) {
+                result.exceptionOrNull()?.let { e { it.localizedMessage } }
```

Comment:
```suggestion
                result.exceptionOrNull()?.let { e { it } }
```

Afaik, Slimber has support for error-logging exceptions without converting them to Strings. I think this is the best way to go here 🙂